### ISSUE TITLE
Adapt emission factors to other countries

### ISSUE THREAD

**Issue by redfrexx** [pledge4future/co2calculator#29]

**Comment by han16nah** [comment]
Possibly useful resources:
- http://efdb.apps.eea.europa.eu/
- https://bilans-ges.ademe.fr/en/accueil/contenu/index/page/emisions_factor_categories/siGras/0 (similar project to ours?)
- https://www.climfoot-project.eu/en/overview-2 (related to the above I think - there is a link to an emission database, but log-in is required) and https://www.climfoot-project.eu/en/overview-4
- https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021 (we already used this for air transport and EV emissions)
- https://www.ipcc-nggip.iges.or.jp/public/2006gl/index.html
- not for free: https://www.iea.org/data-and-statistics/data-product/emissions-factors-2021
- https://www.carbonfootprint.com/docs/2020_07_emissions_factors_sources_for_2020_electricity_v1_3.pdf
- not sure if useful, but defo interesting: https://www.climate-transparency.org/g20-climate-performance/the-climate-transparency-report-2020

**Comment by han16nah** [comment]
ON EMISSION FACTORS FOR **FRANCE**
- **https://data.ademe.fr/datasets/base-carbone%28r%29** (open - I was able to download a csv, maybe we can translate this? seems to be quite extensive)
- https://bilans-ges.ademe.fr/en/accueil/authentification/index/page/downloaddata/siGras/0 (requires log-in, and download seems to require subscription - we have to check, I think the link above is already sufficient anyways?)
- see https://labos1point5.org/static/Justification%20choix%20facteurs%20d%20emission-GES1point5-16062021-site.pdf (we should check this in detail)

WIkipedia article for train emission factors in France: 
- https://fr.wikipedia.org/wiki/%C3%89missions_de_CO2_des_transports_ferroviaires_en_France (we would need to translate and check sources/credibility; seems like ges1point5 also used these?)

French article on different modes of transport and their emission factors:
- https://www.avem.fr/2015/11/12/est-ce-que-le-vae-est-polluant/#

French article/study on e-scooters (and comparison with other modes of transport):
https://media.motorbox.com/media/7/0/2/702801/702801.pdf -> 105.5 g CO2e/km/passenger.

English study on e-scooters:
- https://iopscience.iop.org/article/10.1088/1748-9326/ab2da8/pdf --> 202 g CO2-eq/passenger-mile (1 mile = 1,609344 km --> ~125.5 g CO2e/km/passenger

**Comment by veitu** [comment]
I started to extract and translate emission factors that could be relevant for us from the csv from https://data.ademe.fr/datasets/base-carbone%28r%29. They are in this excel sheet. Feel free to add to it, I will also continue working on it
[base_carbone_select_translate.xlsx](https://github.com/pledge4future/co2calculator/files/8208647/base_carbone_select_translate.xlsx)

**Comment by veitu** [comment]
updated version... Where should we save it?
[base_carbone_select_translate.xlsx](https://github.com/pledge4future/co2calculator/files/8210002/base_carbone_select_translate.xlsx)

**Comment by han16nah** [comment]
> updated version... Where should we save it? [base_carbone_select_translate.xlsx](https://github.com/pledge4future/co2calculator/files/8210002/base_carbone_select_translate.xlsx)

Maybe in [co2calculator](https://github.com/pledge4future/co2calculator)/[data](https://github.com/pledge4future/co2calculator/tree/dev/data)/other_sources/ ?
Could also first be in a new branch, e.g. `ef_france` until we have a more final version? Maybe also as .csv instead of .xlsx? 
I will have a look at the file within the next days :)

**Comment by han16nah** [comment]
> The csv with France emission factors is in the branch `dev-ef_france` in [co2calculator](https://github.com/pledge4future/co2calculator)/[data](https://github.com/pledge4future/co2calculator/tree/dev/data)/other_sources/
> 
> Differences of the France emission factors compared to our Germany emission factors:
> 
>     * long-distance buses are missing
> 
>     * no data for gasoline and diesel cars of different sizes
> 
>     * heating: no data for electric heating, heat pump, and solar heating
> 
>     * no flight data according to passenger class, but different emission factors exist according to plane size
> 
>     * district heating emission factors for about 600 French municipalities are available ("steam coming from a network")
> 
>     * there are emission factors for some other modes of transport that we don't have yet, like e-scooters

Thank you Veit, great work!

Maybe now that we are starting to have different sources, we can eventually create an overview of the differences between the emission factors and discuss them a bit :) might even be solved with a small python script.

**Comment by alexbercik** [comment]
In the process of synthesizing lots of sources for life cycle assessment of emission factors, will share the results soon. But in the meantime wanted to share this resource for UK, which is quite extensive. Contains everything we would need to extend the tool there (heating, electricity, gas, steam, refrigerants, transportation, waste disposal, etc)

https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2020

**Comment by han16nah** [comment]
> In the process of synthesizing lots of sources for life cycle assessment of emission factors, will share the results soon. But in the meantime wanted to share this resource for UK, which is quite extensive. Contains everything we would need to extend the tool there (heating, electricity, gas, steam, refrigerants, transportation, waste disposal, etc)
> 
> https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2020

Yes, we already use the emission factors for air travel from this source :) But true, it is very extensive and definitely an important resource

**Comment by han16nah** [comment]
Here is a nice list of life cycle databases: https://ghgprotocol.org/life-cycle-databases
includes PROBAS, GEMIS and Bilan Carbone but also many many more! very useful @veitu @alexbercik

**Comment by alexbercik** [comment]
I'm currently assembling a list of emission factors for different power grids by location (compiling it from the ghg protocol website, which is great). I came across some terminology, namely the difference between a "location-based" approach and a "market-based" approach. Location-based is the simpler, older way of doing things in IPCC reporting, where for each region a single average emission factor is assigned for a given energy mix. A market-based approach though takes into account that companies can pay extra to ensure that a certain percentage of power comes from renewable sources. The average consumer then that does not pay these extra fees is actually using energy from a "residual-mix", that is the emission factor will be slightly higher since the relative contribution of non-renewable sources will be higher. This residual mix is calculated by explicitly subtracting the green energy contributions ensured these contracts from the overall mix. 

This leads to the question, should we be using residual mixes (when available) in our methodology? Or should we stick to the simpler location-based averages? Thoughts?

**Comment by alexbercik** [comment]
I had a discussion with Veit today. We are leaning towards using the "market-based" approach. Even though ideally we would use a residual mix, this offers some challenges. For example, with a market-based approach we can get emission factors directly from the UN National Inventory Submissions (https://unfccc.int/ghg-inventories-annex-i-parties/2022) which are updated every year and (I believe) are always given in CO2e/kWh, which allows for some standard. This will make updating the factors easy in the future. On the other hand, for residual mixes we have to rely on third-parties. For North America for example, residual mix calculations are done by Green-e* (https://www.green-e.org/residual-mix) and for Europe they are done by the Association of Issuing Bodies* (https://www.aib-net.org/facts/european-residual-mix). While this worked well up until the 2020 release (for 2018 data), in the 2021 report (2019 data) the European mixes were no longer reported in CO2e/kWh, denoted "GWP Direct = Direct onsite Global Warming Potential emissions". Instead, only Direct CO2 emissions were given due to difficulties in updating the advanced emission factors stemming from international trading activities. For that reason it is not entirely clear how to use residual mixes any more. Perhaps another source can be used for Europe? For now it is easier to stick to a market-based approach.

* note that these are the same sources used by the GHG emissions calculation tool developed by Greenhouse Gas Protocol (https://ghgprotocol.org/ghg-emissions-calculation-tool) as of May 2022

**Comment by veitu** [comment]
I put the emission factors I had collected for France into the csv files electricity and transport in the branch dev-emissionfactors (finished). I still have to put in the emission factors for heating, will do that eventually.

**Comment by alexbercik** [comment]
As mentioned in Issue #91 (https://github.com/pledge4future/co2calculator/issues/91), we will add a TODO on the current dev-emissionfactors branch emission factor csv files to add a column indicating whether or not the source is consistent with the values used by the GHG protocol.

**Comment by veitu** [comment]
Electricity
done: energy mix EU countries
missing: solar energy Belgium, Netherlands, Luxembourg, Spain, Portugal
